polybuild/polybuild.py

```diff
@@ -160,6 +160,8 @@ def poly_build(self, argv) -> bool:
             compile_command.append("-I" + self.meta.compiler_dir + "/../cxx_libs/include/c++/v1/")
             compile_command.append("-L" + self.meta.compiler_dir + "/../cxx_libs/lib/")
         for arg in argv[1:]:
+            if arg == "-Wall" or arg == "-Wextra" or arg == "Wno-unused-parameter" or arg == "-Werror":
```

Comment:
Sorry for the drive-by review, but you're missing a `-` before `Wno-unused-parameter` 😉

Reply:
Ayy thanks thanks I missed that :)

Reply:
copy paste issue lol